src/twisted/conch/ssh/service.py

```diff
@@ -30,10 +31,6 @@ def serviceStopped(self):
         or by another service being started
         """
 
-    def logPrefix(self):
```

Comment:
Same thing with `logPrefix` here: the method must be restored (and preferably deprecated).